plugins/modules/notification/mattermost.py

```diff
@@ -61,7 +61,10 @@
     type: str
     description:
       - Url for the message sender's icon.
```

Comment:
```suggestion
      - URL for the message sender's icon.
      - The default is C(https://www.ansible.com/favicon.ico) if neither I(icon_url) or I(icon_emoji) is specified.
```

Reply:
This one is directly related to whatever we decide on in https://github.com/ansible-collections/community.general/pull/5113#discussion_r945141510

Reply:
Change incorporated in my latest commit.